error-messages/error_code_parser_v2.py

```diff
@@ -0,0 +1,43 @@
+import re
```

Comment:
Add some explanation of how to run the script/what it produces as output at the top - see my `get-docs-urls` script in docs-tools as an example